quantumnetworks/systems/base.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, params: Dict[str, Any]) -> None:
             params (dict):
                 key (str): parameter description
                 value (Any): float, list, anything really
-        
+
```

Comment:
Hm, I'm wondering if this is using the black formatter or if you have some other formatter enabled.
Could you please check and reformat with black if necessary?